Help-Them-Keep-Learning.md

```diff
@@ -0,0 +1,64 @@
+## How to Help Attendees Keep Learning
+
+### Just one weekend is not enough, of course.
+
+Installing Clojure stuff on Friday, learning Clojure basics on Saturday, then she's good to go to become a Clojure developer?
+Except for a very few cases, of course, the answer is no.
+As many of you suspect, just one weekend is not enough to start coding Clojure; moreover, it's not enough to keep coding Clojure.
+This document contains ideas for ClojureBridge volunteers to help workshop attendees further after the workshop.
+
+
+### Provide a list of local meetups and/or community info
+
+Since a ClojureBridge workshop is driven by a community in the area, we can assume there is also a local Clojure community in the area.
+Give ClojureBridge attendees local community info, upcoming events or other resources they can take advantage of.
+When you give the info, it's a good idea to hand them a piece of paper.
+Many of the attendees are not familiar with meetup.com, github.com, etc.
+If you just show or say urls of those, they may miss the info.
+
+
+### Invite them to local Clojure events
+
+Clojure developers you have not met yet may attend your ClojureBridge workshop.
+If that is the case, this is a chance to invite them to local Clojure events, such as meetups, and expose them to Clojure technology and community.
+If some of the attendees who are new to Clojure come to a local Clojure event, it means they have made a great first step to getting more involved with Clojure.
+
+
+### Organize follow-up workshop or event
+
+It is a good idea to organize a follow-up workshop or event.
+This may be a short - one or two hours - event like a meetup you do regularly.
+During the follow-up event, walking through the ClojureBridge curriculum or other introductory materials would be good.
+Either lecture style or hack night style will help attendees to keep learning.
+If some folks have been coding a little on their own, starting a project may be an idea - for example, a quil app of their own.
+Since this sort of short event is easy to organize, you can do more than one or even a recurring event.
+
+
+### Follow-up meetups in the past
+
+1. Durham NC - Intro to Clojure at a local Clojure meetup
+2. San Francisco - 
+
+
+### ClojureBridge Alumni mailing list
+
+[https://groups.google.com/forum/#!forum/clojurebridge-alumni](https://groups.google.com/forum/#!forum/clojurebridge-alumni)
```

Comment:
In Markdown one can use angle brackets for a link if using the URL as the link text. In this case:

```diff
-[https://groups.google.com/forum/#!forum/clojurebridge-alumni](https://groups.google.com/forum/#!forum/clojurebridge-alumni)
+<https://groups.google.com/forum/#!forum/clojurebridge-alumni>
```